package.json

```diff
@@ -12,8 +12,8 @@
     "prettier": "npx prettier . --write"
   },
   "dependencies": {
-    "@abgov/react-components": "4.17.0",
-    "@abgov/web-components": "1.17.0",
+    "@abgov/react-components": "4.18.0",
```

Comment:
Can you add the `^` to these versions. Then we don't have to update it all the time.